src/main/web/templates/handlebars/list/partials/tab-item.handlebars

```diff
@@ -1,8 +1,12 @@
 <li class="tab__item width-sm--6">
     {{#if active}}
-        <span class="tab__link tab__link--active">{{name}} ({{nf count alt=0}})</span>
+        <a href="{{href}}?{{#if parameters.q}}q={{parameters.q.0}}{{else}}query={{parameters.query.0}}{{/if}}"
+                aria-current="page" aria-label="{{name}} ({{nf count alt=0}}). Current page.">
+            <span class="tab__link tab__link--active">{{name}} ({{nf count alt=0}})</span>
```

Comment:
Could you actually remove the span and move the class on the anchor? Sorry I missed this the first time.

Reply:
You're right, doesn't seem to be adding anything. Removed now.